pkg-manager/resolve-dependencies/src/resolveDependencies.ts

```diff
@@ -865,10 +875,10 @@ async function resolveDependenciesOfDependency (
     // as an importer separately, and we can rely on that process keeping the
     // importers lockfile catalog snapshots up to date.
     if (catalogLookup != null) {
-      extendedWantedDep.wantedDependency.bareSpecifier = getCatalogReplacementBareSpecifier(
-        catalogLookup,
-        ctx.wantedLockfile,
-        extendedWantedDep.wantedDependency)
+      const existingVersion = getCatalogExistingVersionFromSnapshot(catalogLookup, ctx.wantedLockfile, extendedWantedDep.wantedDependency)
```

Comment:
I think there's no need in this new variable. It doesn't provide additional context as the function name is the same.